freud/environment.pyx

```diff
@@ -149,14 +149,14 @@ cdef class BondOrder:
         Args:
             box (:class:`freud.box.Box`):
                 Simulation box.
-            ref_points ((:math:`N_{particles}`, 3) :class:`numpy.ndarray`):
+            ref_points ((:math:`N_{ref_points}`, 3) :class:`numpy.ndarray`):
```

Comment:
I think the docs will render this wrong unless you escape the underscore. Please check.